src/main/java/org/gelecekbilimde/scienceplatform/auth/model/response/UserRoleResponse.java

```diff
@@ -0,0 +1,14 @@
+package org.gelecekbilimde.scienceplatform.auth.model.response;
+
+import lombok.*;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class UserRoleResponse {
+	private String userId;
+	private String userEmail;
+	private String roleName;
```

Comment:
Nested Class'lar oluşturup bu field'ları oraya ekleyelim, bir de response dönmek zorunda mıyız?
```suggestion
	private User user;
	private Role role;

    ... nested classes
```

Reply:
Response dönmesek de olabilir sadece basit bir rol değişim işlemi. Frontendde eğer Ok response döndü ise isteğiniz başarıyla gönderilmiştir ya da kullanıcı rolü güncellenmiştir eklenebilir.